pkg/serverless/trace/inferredspan/span_enrichment.go

```diff
@@ -113,3 +143,15 @@ func isAsyncEvent(attributes EventKeys) bool {
 func calculateStartTime(epoch int64) int64 {
 	return (epoch / 1000) * 1e9
```

Comment:
this is strange since I've updated this, could you merge main in your branch just to be sure?